openvmm/hvlite_entry/src/lib.rs

```diff
@@ -496,7 +504,7 @@ fn vm_config_from_command_line(
             if !opt.no_alias_map {
                 anyhow::bail!("must specify --no-alias-map to offer NICs to VTL2");
             }
-            let mana = mana_nics[2].get_or_insert_with(|| {
+            let mana = mana_nics[openhcl_vtl as usize].get_or_insert_with(|| {
```

Comment:
VtlArray?

Reply:
I considered it. But ehh it's OK for now.